src/sleep/meson.build

```diff
@@ -3,7 +3,7 @@
 executables += [
         libexec_template + {
                 'name' : 'systemd-sleep',
-                'sources' : files('sleep.c'),
+                'sources' : files('sleep.c', 'battery.c'),
```

Comment:
Please do one-per-line. There are only two other places which do a multi-item one-line `files`, and we should probably change those too ;)

Reply:
-> #29282